### ISSUE TITLE
Build unsigned APK for fast testing it

### ISSUE THREAD

**Issue by objestik** [Corsage/Masterani#6]
Could you please build the app and send it to me for testing it? :) 
I'm currently away from my working computer for a couple of days, and would like to test the app on my Sony Xperia phone

**Comment by Corsage** [comment]
There is a build available under the releases: https://github.com/Corsage/Masterani/releases

The direct link is https://github.com/Corsage/Masterani/releases/download/v2.0-alpha/masterani-debug-v2.apk

Thanks for testing it out! :)

**Comment by objeen** [comment]
The app crashes when you click on a episode of a show.

**Comment by Corsage** [comment]
What show and episode? Or does this for all of them?

**Comment by objeen** [comment]
Most of the show don't play and the app crashes for e.g Boruto, One piece other popular today or currently watching. 

Also can you please put a resolution option and also the video only play 720p only it takes half of my screen using Samsung S9.

Could you also put a play option so I can play the video with the video player of what I want to play it with VLC, MX player. Add a Download option if you can.

**Comment by Corsage** [comment]
Yeah, just checked it crashes for me as well. I will look into this, thanks for your help!

The next version of this app will have all of those ideas you just posted ;)

Thank you for being patient haha 👍

**Comment by objeen** [comment]
Can you please build a new updated version  of the app and send it for testing.

**Comment by mohalobaidi** [comment]
It doesn't work with Samsung S9 :/

**Comment by objeen** [comment]
Yes I know that that's why I told the developer the issue and he is fixing the issues that I had with the app.
